code/_onclick/click.dm

```diff
@@ -280,7 +280,7 @@
 	if(!specific_direction)
 		specific_direction = direction
 
-	facedir(direction, specific_direction)
+	faceDir(direction, specific_direction)
```

Comment:
so you made it camelCase instead of snake_case which is our standard?

needs to be face_dir

Reply:
riiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiight

![image](https://user-images.githubusercontent.com/53100513/209213172-e06205c5-70e0-4e1c-8567-938b8c06c0ca.png)

Reply:
done